contributors.json

```diff
@@ -68,6 +68,11 @@
     "name": "Ana Carolina Arellano",
     "website": "https://anacarolinaarellano.dev/"
   },
+  "kontesanjana":{
```

Comment:
Order by github user name in ascedning order

Reply:
yeah, thank you.